packages/tsdocs/README.md

```diff
@@ -0,0 +1,67 @@
+# @loopback/tsdocs
+
+This module provides [API docs](https://github.com/Microsoft/tsdoc) generation
+for `@loopback/*` packages.
+
+It's built on top of https://api-extractor.com/:
+
+- https://github.com/Microsoft/web-build-tools/tree/master/apps/api-extractor
+- https://github.com/Microsoft/web-build-tools/tree/master/apps/api-documenter
+
+## Basic Use
+
+### Build api reports and doc models
+
+```sh
+npm run extract-apidocs
+```
+
+The command above will traverse all TypeScript packages in the monorepo and run
+`api-extractor` to generate api reports and doc models into
+`loopback-next/docs/apidocs`:
+
+- `reports`: api reports
+- `reports-temp`: temporary api reports
+- `models`: doc models
+
+### Generate api docs as markdown files
+
+```sh
+npm run document-apidocs
+```
+
+It runs `api-documenter` to generate markdown files into
+`loopback-next/docs/site/apidocs`.
```

Comment:
I didn't see this directory in `.gitignore`. Please ensure that all generated files (reports, reports-temp, any Markdown files) are excluded from git.